apps/communications/contacts/elements/settings.html

```diff
@@ -34,28 +34,6 @@ <h1 data-l10n-id="settings"></h1>
               </button>
             </li>
           </ul>
-          <gaia-subheader id="fb-header" skin="organic">
-            <span data-l10n-id="facebook">Facebook</span>
-          </gaia-subheader>
-          <ul id="settingsFb" role="presentation" data-type="list" data-state="logged-out">
-            <li class="fb-item" role="checkbox">
-              <p data-l10n-id="facebookSwitchMsg">Sync friends</p>
```

Comment:
Yes, definitely. If you're not using those strings anywhere else, remove them from resource files.